languages/elixir/exercises/concept/anonymous-functions/.docs/introduction.md

```diff
@@ -0,0 +1,62 @@
+Functions are treated as first class citizens in Elixir. This means functions:
+
+- Can be assigned to variables.
+- Can be passed around like data as arguments and return values.
+- Can be created dynamically.
+
+Anonymous functions, in contrast to named functions, don't have a static reference available to them, they are only available if they are assigned to a variable or immediately invoked.
+
+We might use anonymous functions to:
+
+- Hide data using lexical scope (also known as a closure).
+- Dynamically create functions at run-time.
+
+Anonymous function start with the reserved word `fn`, the parameters are separated from the body of the function with the `->` token, and they are finished with an `end`. As with named functions, the last expression in the function is _implicitly returned_ to the calling function.
+
+To invoke a function reference, you must use a `.` between the reference variable and the list of parameters:
+
+```elixir
+function_variable = fn param ->
+  param + 1
+end
+
+function_variable.(1)
+# => 2
+```
+
+You can even use short hand capture notation to make this more concise:
+
+```elixir
+variable = &(&1 + 1)
+
+variable.(1)
+# => 2
+```
+
+#### Bitwise operations
+
+Bitwise operations are performed on the binary representation of data. Performing a _bitwise-and_ on the binary numbers, `1` and `0`, results in `0` because the bits differ. In contrast _bitwise-or_ on the binary numbers, `1` and `0`, results in `1`.
+
+It may seem that these operations resemble boolean logic. The difference is bitwise operations are performed on a binary number of any length, not limited to 1-bit.
+
+```text
+              1010100
+(bitwise-and) 1101010
+              -------
+              1000000
+
+
+              1010100
+ (bitwise-or) 1101010
+              -------
+              1111110
+
+              1010100
+(bitwise-xor) 1101010
+              -------
+              0111110
+```
+
+In these examples, we can see the operation being performed on the pair of binary numbers, where the each digit in the top row is applied one-by-one to the digit in the bottom row to obtain the result.
+
+Elixir supports these operations with functions found in the _Bitwise module_.
```

Comment:
I think the last line here is the most important one. The rest could probably be omitted, as we don't have to explain bitwise operators. Does that make sense?

Reply:
Do you mean leave just line 62? I wasn't sure how much I could assume someone to know.

Reply:
This is a tricky part. I don't think we need to explain how bitwise logic works, as long as we properly name it as such in order for students to be able to google it. 

This is how the C# example introduces bitwise operators: https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/flag-enums/.docs/introduction.md

For this exercise, students actually don't _have_ to know how bitwise operations work, but just know that they exist and where to find them :)

Reply:
Ok, I think I understand the goal. I can adapt what you have for c# for this purpose.

Reply:
Excellent!